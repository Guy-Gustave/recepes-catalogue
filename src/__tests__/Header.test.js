
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <div className="App">
        <BrowserRouter>
          <h2>Recepes calalogue</h2>
        </BrowserRouter>
      </div>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
