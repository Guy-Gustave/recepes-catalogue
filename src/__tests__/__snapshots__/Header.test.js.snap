// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <h2>
    Recepes calalogue
  </h2>
</div>
`;
